### PR TITLE
Working on 'sysfetch' command

### PR DESCRIPTION
**Pull request by ghost**
Will be updated when learned how to print something

### CODE REVIEW

File: apps/terminal.asm [hunk 1]
```diff
@@ -226,7 +234,7 @@ main:
 .ls.error.STRING: db "Error while loading disk data",0x00
 
 DATA:
-.PROMPT_STR: db "</> ",0x00
+.PROMPT_STR: db "C:/> ",0x00
```
Comment (ghost) [inline apps/terminal.asm]: This can be edited but this looks better

**Comment by AtieP** [comment]
What's sysfetch?

**Review by ghost — COMMENTED** [review]
> What's sysfetch?

I'm working on it ( Waiting you to tell me how to print something ). It will be printing system information.

Example output;
`         Operating System
Name : FutureDOS
Version : Pre-Alpha
Release : 153
Build : 149
         Kernel
Name : FutureDOS
Version : Pre-Alpha
Release : 153
Build : 149`
And more information...

**Comment by AtieP** [comment]
Ah, okay. Well, I am planning in a near future to add `get_system_info` syscalls, so for now you shouldn't do anything yet.

**Comment by AtieP** [comment]
until they're implemented ofc

**Comment by AtieP** [comment]
and BTW you should reset hard and push force to delete your latest commits because they don't add anything to the operating system, it's just a command that does nothing.

**Comment by ghost** [comment]
Can accept the pr? I'll update it and then send a new pull request ( BTW IM STILL WAITING YOU TO TELL ME HOW CAN I PRINT SOMETHING )

**Comment by AtieP** [comment]
> Can accept the pr?

No, sorry, we only accept complete PRs.

> ( BTW IM STILL WAITING YOU TO TELL ME HOW CAN I PRINT SOMETHING )

I already told you on the Discord server. Read `docs/syscall.md` and look up for `puts` (ah = `0x06`).

Edit: `syscall.asm` -> `syscall.md`

**Pull request closed without merging**